### PR TITLE
[reactions] alcohol and biomolecules

### PR DESCRIPTION
**Pull request by krishsharma0413**
Most reactions of alcohols, phenol, aldehyde and ketones are missing.
If you are working on the UI then let me work on adding reactions 🫂

### CODE REVIEW

**Comment by CodeWithSwastik** [comment]
Sure you can help with the reactions! Also I don't think adding biomolecules right now is a good choice so let's leave that out for now.

File: reactions.json [hunk 1]
```diff
@@ -122,12 +122,18 @@
     "HCl": "chloroethane",
     "HBr": "bromoethane",
     "H2O / dil H2SO4": "ethanol",
-    "O3 / Zn dust / H2O": "methanal"
+    "O3 / Zn dust / H2O": "methanal",
+    "B2H6": "ethan-1-ol"
```
Comment (CodeWithSwastik) [inline reactions.json]: ethan-1-ol is redundant, just ethanol is fine

Comment (krishsharma0413) [reply reactions.json]: I did that because i saw `propan-1-ol` on line 71 & 72
Do you want me to change those as well?

**Comment by krishsharma0413** [comment]
alright sounds ok to me

File: reactions.json (hunk 1 above)
Comment (sleepbot-zapp) [reply reactions.json]: yea i took care of that

Comment (krishsharma0413) [reply reactions.json]: Yeah i added another commit to remove 1-ol from it check latest commit @sleepbot-code

Comment (krishsharma0413) [reply reactions.json]: I think its ok to be merged(?)

**Comment by CodeWithSwastik** [comment]
On propane derivatives and higher the 1 is not redundant. It is required to avoid ambiguity.  Please make the required changes. 

For methane, ethane the substituent can only be on position 1 so it makes no sense to specify it. For propane the substituent can be either on the first or second carbon, it then becomes highly important to specify the position.

**Comment by krishsharma0413** [comment]
But if no number is given then its self explanatory than it have to be 1 only

**Comment by CodeWithSwastik** [comment]
> But if no number is given then its self explanatory than it have to be 1 only

I'm pretty sure it is required and not just assumed. Anyway it is way better to be clear with compound names.

**Comment by krishsharma0413** [comment]
Then i will add it to all then

**Comment by krishsharma0413** [comment]
or nvm

**Review by CodeWithSwastik — APPROVED** [review]

**Comment by sleepbot-zapp** [comment]
> But if no number is given then its self explanatory than it have to be 1 only
>  yes
> I'm pretty sure it is required and not just assumed. Anyway it is way better to be clear with compound names.
> uh yes good to be clear with compound names, but i am pretty sure that if the locant is 1 on a derivative of hydrocarbon with any number of carbons. 1 is redundant.

**Pull request merged**